src/memory.rs

```diff
@@ -55,6 +55,9 @@ impl Pointer {
     pub fn points_to_zst(&self) -> bool {
         self.alloc_id == ZST_ALLOC_ID
     }
+    pub fn to_int(&self) -> usize {
+        self.offset
```

Comment:
Then we need to create a value ptr alloc, because we want to be able to offset integer pointers